src/logout/components/Logout/index.js

```diff
@@ -1,16 +1,26 @@
-import { compose } from 'recompose';
+import { compose, withProps } from 'recompose';
 import { withActions } from 'spunky';
-
+import { connect } from 'react-redux';
 import accountActions from 'shared/actions/accountActions';
-
 import Logout from './Logout';
 import withLogout from '../../hocs/withLogout';
 
+import { emptyAll } from '../../../browser/actions/requestsActions';
+
 const mapActionsToProps = ({ reset }) => ({
-  logout: reset
+  reset
 });
 
 export default compose(
   withActions(accountActions, mapActionsToProps),
-  withLogout((state, { history }) => history.push('/login'))
+  connect(null, (dispatch) => ({
```

Comment:
Last piece of feedback. To be consistent with our other files, can this function be lifted into a `mapDispatchToProps` const before it's used here?

Reply:
should be good now 👍